source/config/configFlags.py

```diff
@@ -251,3 +251,20 @@ def _displayStringLabels(self):
 			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
 			self.SPEECH_AND_BRAILLE: _("Speech and braille"),
 		}
+
+
+class ParagraphStartMarker(DisplayStringStrEnum):
+	NONE = ""
+	SPACE = " "
+	PILCROW = "¶"
+
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: This is a label for a paragraph start marker.
+			self.NONE: _("No paragraph start marker (default)"),
+			# Translators: This is a label for a paragraph start marker.
+			self.SPACE: _("Double space (  )"),
+			# Translators: This is a label for a paragraph start marker.
```

Comment:
This might help translators understand the symbol better and know how to translate it.

```suggestion
			# Translators: This is a label for a paragraph start marker.
			# Pilcrow is a symbol also known as "paragraph symbol" or "paragraph marker".
			# Ensure this is consistent with other strings with the context "paragraphMarker".
```